addons/l10n_it_edi/models/ir_mail_server.py

```diff
@@ -136,11 +136,11 @@ def _attachment_invoice(self, msg_txt):
                     self._message_receipt_invoice(split_underscore[1], attachment)
                 else:
                     att_filename = attachment.fname
-                    match = re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(xml.p7m|xml))", att_filename)
+                    match = re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.((?i:xml.p7m)|(?i:xml)))", att_filename)
```

Comment:
this works ok, but again it could be
`(?i:xml.p7m|xml)`

Reply:
simpler idd.  I pushed like that.